extension/content.js

```diff
@@ -189,17 +189,23 @@ function infinitelyMore() {
 function addBlameParentLinks() {
 	$('.blame-sha').each((index, commitLink) => {
```

Comment:
You need to make sure you're not selecting the links you just shoved into the DOM:
```js
$('.blame-sha').not('.js-blame-parent').each((index, commitLink) => {
```

Without this fix there's an error:
```
Uncaught TypeError: Cannot read property '0' of null                             content.js:198
```